conda/environment-dev.yml

```diff
@@ -8,6 +8,7 @@ dependencies:
   - markdown=3.3
   - mkdocs=1.1.2
   - mkdocs-material=5.5.3
+  - mkdocs-material-extensions>=1.0
```

Comment:
Could you tell us how this was an issue (i.e. how you reached the need for this dependency)? I don't recall having to explicitly add this before.

Reply:
Sure. After I installed the full library by conda, when I ran `mkdocs serve`, I got an error in mkdocs:
```
...
  File "/net/nfs2.prior/tianwein/miniconda3/envs/allenact/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2472, in require
    items = working_set.resolve(reqs, env, installer, extras=self.extras)
  File "/net/nfs2.prior/tianwein/miniconda3/envs/allenact/lib/python3.6/site-packages/pkg_resources/__init__.py", line 772, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'mkdocs-material-extensions>=1.0' distribution was not found and is required by the application
```
Then I install the `mkdocs-material-extensions` to solve this issue.

Reply:
Ok, thanks